client/index.js

```diff
@@ -30,7 +30,7 @@ const supportsHistory = 'pushState' in window.history;
 
 // Get any rehydrateState for the async components.
 // eslint-disable-next-line no-underscore-dangle
-const asyncComponentsRehydrateState = window.__ASYNC_COMPONENTS_REHYDRATE_STATE__;
+const asyncComponentsRehydrateState = window.__JOBS_STATE__;
```

Comment:
Should this part in [server/middleware/reactApplication/ServerHTML.js](https://github.com/ctrlplusb/react-universally/blob/master/server/middleware/reactApplication/ServerHTML.js#L69) also reflect this?
```
 ifElse(asyncComponentsState)(() =>
      inlineScript(
        `window.__ASYNC_COMPONENTS_REHYDRATE_STATE__=${serialize(asyncComponentsState)};`,
      ),
    ),
```
also reflect this change?